instrumentation/github.com/Shopify/sarama/consumer_group_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sarama
+
+// TODO: add test for consumer group
+// Currently, sarama does not have a mock consumer group, so it's hard to
+// write a unit test.
+// Related PR: https://github.com/Shopify/sarama/pull/1750
```

Comment:
Can you create an issue for this repo to track this please?

Reply:
I created this issue. #163